services/QuillLMS/app/services/independent_practice_packs_assigner.rb

```diff
@@ -19,8 +19,8 @@ def initialize(
     user:
   )
     @assigning_all_recommendations = assigning_all_recommendations
-    @classroom_id = classroom_id
-    @diagnostic_activity_id = diagnostic_activity_id
+    @classroom_id = classroom_id.to_i
+    @diagnostic_activity_id = diagnostic_activity_id.to_i
```

Comment:
Could this be `nil` for a non-diagnostic? If so, let's do a `nil` check.

Reply:
At this point, pack sequences are only constructed via the diagnostic activity route so this can be assumed to be non-nill.